ambry-coordinator/src/test/java/com.github.ambry.coordinator/MockBlockingChannel.java

```diff
@@ -152,13 +156,14 @@ public void send(Send send)
             break;
 
           case Blob:
-            MockDataNode.BlobOutputAndError bdae = mockDataNode.getData(blobId);
+            MockDataNode.BlobDataAndError bdae = mockDataNode.getData(blobId);
             getResponseErrorCode = bdae.getError();
             if (getResponseErrorCode == ServerErrorCode.No_Error) {
-              BlobOutput blobData = bdae.getBlobOutput();
-              byteBufferSize = (int) MessageFormatRecord.Blob_Format_V1.getBlobRecordSize(blobData.getSize());
+              BlobData blobData = bdae.getBlobData();
+              byteBufferSize = (int) MessageFormatRecord.Blob_Format_V2.getBlobRecordSize(blobData.getSize());
               byteBuffer = ByteBuffer.allocate(byteBufferSize);
-              MessageFormatRecord.Blob_Format_V1.serializePartialBlobRecord(byteBuffer, blobData.getSize());
+              MessageFormatRecord.Blob_Format_V2
```

Comment:
Somehow I feel that having BlobData would be more intuitive rather than having BlobOutput. Else, we have to add comments everywhere stating that this is legacy code and used only for simple blobs(or old style blobs) and hard to maintain. Making it BlobData makes is more intuitive and don't need of any comments as getBlob() in general returns BlobData